utils/random.ts

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
consider using a package ie https://github.com/klughammer/node-randomstring instead of writing our custom implementation

Reply:
I will if we'll need more complex stuff, so far it's easy job and I'd rather not add another dependency

Reply:
I'll do that instead, your code is unsafely using Math.random, we need something better:
https://github.com/klughammer/node-randomstring/blob/master/lib/randomstring.js#L9

more code = more bugs, please use randomstring

Reply:
bonus: is already included: https://github.com/pagopa/io-functions-admin/blob/master/package.json#L61

Reply:
random string isn't working for us as it doesn't ensure the string will contain some for character for any character group. I'd need to fin another lib to include

Reply:
where does this requirement come from ? you can concatenate more random strings if you really need this (do we?)

Reply:
"Passwords are needed to be strong". What does _strong_ mean? I don't know. My interpretation was:
* 18 characters
* at least one uppercase letter
* at least one lowercase letter
* at least one number
* at least one symbol
* no pattern
Shold we check with security guys?

Reply:
according to pietro we don't need a constraint on the number of symbols for each set, so a 18 character string with one character set that has upper/lower case alphanumeric characters and special symbols is ok